### DISCUSSION TITLE
An error occurred with your deployment and 'The upstream server is timing out'

### DISCUSSION THREAD

**Discussion by 0hq** [supabase/supabase#2545] [Questions] [answered]
Have a large DB (2.5 M rows) and trying to query by unique id and timestamp.
Any medium request creates An error occurred with your deployment in the SQL editor and 'The upstream server is timing out' from supabase-js.

Seems like a supabase error since query is formatted correctly and the request is what's failing. Queries work fine for other smaller tables.

**Comment by inian** [comment, marked as answer]
Marking this as answered since there as an issue for this as well - https://github.com/supabase/supabase/issues/2547. Lets continue the conversation there
